src/containerapp/azext_containerapp/_help.py

```diff
@@ -955,7 +960,11 @@
       text: |
           az containerapp update -n my-containerapp -g MyResourceGroup \\
               --enable-java-metrics
-    - name: Update a container app without java metrics enabled
+    - name: Update a container app with java agent enabled
+      text: |
+          az containerapp update -n my-containerapp -g MyResourceGroup \\
+              --enable-java-agent
+    - name: Update a container app without java metrics/java agent enabled
```

Comment:
--runtime generic will unset all java settings, including java agent settings? should reword this sample